src/main/java/conan/commands/IsInstalledCommand.java

```diff
@@ -23,20 +16,8 @@
      * @return true iff Conan executable exists in env path.
      */
     public static boolean isInstalled(Project project) {
-        ConanCommandBase command = new ConanCommandBase(project);
-
-        ProcessHandler processHandler;
-        try {
-            processHandler = new OSProcessHandler(command.args);
-            processHandler.startNotify();
-            boolean ret = processHandler.waitFor();
-            if (!ret) {
-                log(logger, "Conan is not installed", "", NotificationType.INFORMATION);
-            }
-            return ret;
-        } catch (ExecutionException e) {
-            log(logger, e.getMessage(), Arrays.toString(e.getStackTrace()), NotificationType.INFORMATION);
-            return false;
-        }
+        Version command = new Version(project);
+        String version = command.run_sync();
+        return version != null && !StringUtils.isBlank(version);
```

Comment:
```suggestion
        return !StringUtils.isBlank(version);
```

``isBlank`` already checks for null